template.tpl

```diff
@@ -180,6 +182,13 @@ ___TEMPLATE_PARAMETERS___
         "displayName": "Segment",
         "simpleValueType": true,
         "canBeEmptyString": true
+      },
+      {
+        "type": "CHECKBOX",
+        "name": "spa",
+        "checkboxText": "Single Page Application",
+        "simpleValueType": true,
+        "help": "Check this box if your website is a single-page application (e.g., it uses a framework like React, Angular, or Vue.js). If you\u0027re not sure, leave this unchecked."
```

Comment:
🟢 _(nit)_: technically we should use _id est_ (i.e.) here.

Reply:
👍